react/reducer/index.ts

```diff
@@ -33,6 +34,17 @@ function reducer(
       }
     }
 
+    case 'SELECT_COLOR_VARIATION': {
```

Comment:
Use the term IMAGE_VARIATION, because it could be something else.

Reply:
CADE SUGGESTION